dockstore_launcher_config/compose.config

```diff
@@ -17,6 +17,8 @@
 "DISCOURSE_URL":"foobar",
 "DOCKSTORE_DBPASSWORD":"replaceme",
 "DOCKSTORE_VERSION":"develop",
+"DOCKSTORE_ZENODO_ACCESS_TOKEN":"",
```

Comment:
Empty strings because if we forget to add these to the secrets manager, then the webservice will see the empty strings and skip DOI auto creation and adding it to the Dockstore community